scripts/scripts.js

```diff
@@ -362,6 +362,10 @@ function getDomainInfo(hostname) {
 
 function pushPageLoadToDataLayer() {
   const { hostname } = window.location;
+  if (!hostname) {
```

Comment:
I expect the hostname to be not null. Rather `main--bitdefender--hlxsites.hlx.page`?

Reply:
in case of sidekick, when loading the page, the `window.localtion.hostname` is an empty string

Reply:
ack!